src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const list = document.querySelector('.gallery__list');
+const picture = document.querySelector('.gallery__large-img');
+
+list.addEventListener('click', (occasion) => {
```

Comment:
```suggestion
list.addEventListener('click', (event) => {
```

please use common names for such variables